tests/trainer/optimization/test_manual_optimization.py

```diff
@@ -68,8 +68,9 @@ def training_step(self, batch, batch_idx):
             assert torch.all(self.layer.weight.grad == 0)
 
         def training_epoch_end(self, outputs) -> None:
-            # outputs should be an array with an entry per optimizer
-            assert len(outputs) == 2
+            # outputs is empty as training_step does not return
+            # and it is not automatic optimization
+            assert len(outputs) == 0
 
```

Comment:
removing the handling reveals this test errors, like manual optimization (when training step does not return).

Reply:
great find! this broad try/catch was suppressing existing errors in our tests